server/resources/commandsV1.json

```diff
@@ -0,0 +1,530 @@
+[
```

Comment:
You can also remove `server/resources/commands.json` in this PR too

Reply:
I've removed it.